ddprof-ffi/src/exporter.rs

```diff
@@ -16,16 +17,16 @@ use std::str::FromStr;
 #[repr(C)]
 pub enum SendResult {
     HttpResponse(HttpStatus),
-    Err(crate::Vec<u8>),
+    Err(ddcommon_ffi::Vec<u8>),
 }
 
 #[repr(C)]
 pub enum NewProfileExporterV3Result {
     Ok(*mut ProfileExporterV3),
-    Err(crate::Vec<u8>),
+    Err(ddcommon_ffi::Vec<u8>),
 }
 
-#[export_name = "ddprof_ffi_NewProfileExporterV3Result_drop"]
+#[export_name = "ddog_NewProfileExporterV3Result_drop"]
```

Comment:
@morrisonlevi not for this PR but this got me thinking -- since we're still iterating on the API and breaking it all the time, did we add the `V3` suffix here too early?

Would you be against me submitting a PR removing the `V3` suffixes from the API names?

Looking at what's needed to move to the `v2.4` intake (where `v3` here was actually renamed `v1.3` in our internal specs), I don't think that will need changing the ffi APIs, only the libdatadog implementation, and anyway we wouldn't need to support both at the same time because the agent just proxies stuff.

Happy to talk more, just thought this was a nice time to bring it up since we're renaming these APIs again anyway so all the time so far we called it v3 it wasn't that helpful lol, we never made use of it, and broke the API anyway :P

Reply:
Yeah, V3 was meant for the v1.3 intake format/protocol. Not sure what we need to do to support both 1.3 and 2.4 and how that's negotiated, but we can figure that out when we need it and not earlier.